frontend/packages/console-shared/src/components/formik-fields/field-types.ts

```diff
@@ -67,6 +67,10 @@ export interface MultiColumnFieldProps extends FieldProps {
   children: React.ReactNode;
 }
 
+export interface YAMLEditorFieldProps extends FieldProps {
+  onChange?: (value: string) => void;
+}
```

Comment:
Why do these prop types keep going into this file instead of co-located in the component file itself?

Reply:
Some of the components such as `DropdownField` and `NSDropdownField` share prop types along with extending the base `FieldProps`. That is why all the field props are there in one file.